src/Entity/Adherent.php

```diff
@@ -516,6 +529,7 @@ public function updateMembership(MembershipRequest $membership, PostAddress $pos
         $this->phone = $membership->getPhone();
         $this->comEmail = $membership->comEmail;
         $this->comMobile = $membership->comMobile;
+        $this->adherent = true;
```

Comment:
🤔 So editing a profile is enough to be adherent?

Reply:
No, finalize your registration in en-marche is enough

Reply:
So this should be removed, this makes it true for any edition (update) of the profile.